Gemfile.lock

```diff
@@ -634,6 +634,7 @@ DEPENDENCIES
   email_address (~> 0.1.11)
   factory_bot_rails
   faker
+  foundation_emails
```

Comment:
Wonder if we should we get rid of **premailer** or **inky** as we now have multiple gems that all seem to serve the same purpose?

Reply:
*premailer* and *inky* do different things. I believe we use premailer within our ad rendering don't we? foundation_emails is the updated version of inky I believe. This can be removed though